analytics_data_api/v0/tests/views/test_learners.py

```diff
@@ -641,12 +641,16 @@ def test_csv_fields(self, fields, valid_fields):
 
 
 @ddt.ddt
-class CourseLearnerMetadataTests(DemoCourseMixin, VerifyCourseIdMixin,
-                                 LearnerAPITestMixin, TestCaseWithAuthentication):
+class CourseLearnerMetadataTests(VerifyCourseIdMixin, LearnerAPITestMixin, TestCaseWithAuthentication,):
     """
     Tests for the course learner metadata endpoint.
     """
 
+    @classmethod
+    def setUpClass(cls):
+        super(CourseLearnerMetadataTests, cls).setUpClass()
+        cls.course_id = 'edX/DemoX/Demo_Course'
```

Comment:
Is there a reason you can't use `CourseSamples.course_ids` here like you were doing everywhere else?